### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by MuhammadAliKhalid** [community/community#12812] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

**Comment by davecheney** [comment, marked as answer]
@MuhammadAliKhalid i'm sorry, you are still on the waitlist.  Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Comment by ghost** [comment]
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

help me for that and please approve my account on copilot waiting
